From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Wed, 11 Sep 2019 18:11:19 +0900
Subject: fileio: update warning message

(cherry picked from commit 0f93577647cef5f683d83e89cc63f6124885e708)
(cherry picked from commit 3e1267cf50716ce4f737e6c07294b571a634a512)
---
 src/basic/fileio.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/basic/fileio.c b/src/basic/fileio.c
index 623e43e..a9c0fd2 100644
--- a/src/basic/fileio.c
+++ b/src/basic/fileio.c
@@ -930,10 +930,10 @@ int warn_file_is_world_accessible(const char *filename, struct stat *st, const c
 
         if (unit)
                 log_syntax(unit, LOG_WARNING, filename, line, 0,
-                           "%s has %04o mode that is too permissive, please adjust the access mode.",
+                           "%s has %04o mode that is too permissive, please adjust the ownership and access mode.",
                            filename, st->st_mode & 07777);
         else
-                log_warning("%s has %04o mode that is too permissive, please adjust the access mode.",
+                log_warning("%s has %04o mode that is too permissive, please adjust the ownership and access mode.",
                             filename, st->st_mode & 07777);
         return 0;
 }
